example/sample1/federatedreplicaset-override.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: core.federation.k8s.io/v1alpha1
+  kind: FederatedReplicaSetOverride 
+  metadata:
+    name: test-replicaset
+    namespace: test-namespace
+  spec:
+    overrides:
+    - clusterName: cluster2
+      replicas: 5
```

Comment:
(No action required) Same comment about replicas, the test fixtures now use the minimum (`replicas: 1` for template, `replicas: 2` for override).